vyperlang/vyper
# add CLI flag to disable optimizer

### What I did
Fix #2516 

### How I did it
Add no-optimize flag, thread through the codebase and disable LLL+assembly optimizations

### How to verify it
Build, compile something with `vyper --no-optimize -f ir foo.vy` and `vyper --no-optimize -f asm foo.vy` and see the difference.

### Description for the changelog
Add `--no-optimize` flag to the CLI

### Cute Animal Picture

![Put a link to a cute animal picture inside the parenthesis-->]()
